apps/ex_nvr/lib/ex_nvr/devices/camera_client/milesight.ex

```diff
@@ -0,0 +1,117 @@
+defmodule ExNVR.Devices.CameraClient.Milesight do
+  @moduledoc """
+  Client for Milesight camera
+  """
+
+  require Logger
+
+  alias ExNVR.HTTP
+
+  @lpr_path "/cgi-bin/operator/operator.cgi?action=get.lpr.lastdata&format=inf"
```

Comment:
I think in milesight you can set the `format` query param to `json`. Please check that.